apps/antalmanac/src/components/PatchNotes.tsx

```diff
@@ -1,73 +1,72 @@
-import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@material-ui/core';
-import { useEffect, useState } from 'react';
+import { useCallback, useState } from 'react';
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle } from '@mui/material';
+import { MuiMarkdown } from '$components/MuiMarkdown';
```

Comment:
I like this change but would've been easier if it was it's own PR. Like if you opened a PR that _only_ included this markdown rendering for the patch notes, it would be reviewed and merged in like 10 minutes instead of needing to wait for the rest of this stuff to get reviewed.

Reply:
Not really a defense: I didn't know this was going to be in the PR, I just Googled it and thought it would be fun to add lol

Reply:
Eh, it's fine this tim,e but in the future if you just checkout back to main before adding random stuff like this, it'll get merged in way faster.